.env

```diff
@@ -28,6 +28,10 @@ LDAP_GROUP_SEARCH_ATTRIBUTES=mail,uid
 LDAP_EMAIL_FIELDNAME=mail
 LDAP_ADMIN_GROUP=admin_staff
 
+NEXT_PUBLIC_GOOGLE_ENABLED=false
```

Comment:
could you also open a small PR in the backend too to place the env vars? Both in `.env` and `docker-compose` mapping. We can merge that one first and then update the frontend (general approach when we want to add new feature and env vars)